docs/Documentation/Drivers/Image/RunVIC.md

```diff
@@ -48,11 +51,23 @@ At the command prompt, type:
 
 where `global_parameter_filename` = name of the global parameter file corresponding to your project.
 
-To run VIC image driver using multiple processors, type the following instead:
+The VIC image driver can be run using parallel processing using MPI and/or OpenMP.
+
+!!! Note
+    Users are encouraged to consult their system administrator for assistance in configuring the VIC image driver for parallel processing applications.
+
+To run VIC image driver using multiple processors using MPI, type the following instead:
 
         mpiexec -np $n_proc ./vic_image.exe -g global_parameter_filename.txt
 
-where `n_proc` = number of processors to be used
+where `n_proc` = number of processors to be used. *Note that different MPI implementations may use different names for the MPI executable such as: `mpirun`, `mpiexec_mpt`, or `mpiexec.hydra`*.
+
```

Comment:
I don't think the second asterisk is needed

Reply:
I think it is to close the *italics* block